subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -178,23 +200,42 @@ uint8_t ll_big_test_create(uint8_t big_handle, uint8_t adv_handle,
 
 uint8_t ll_big_terminate(uint8_t big_handle, uint8_t reason)
 {
-	struct ll_adv_iso *adv_iso;
+	struct lll_adv_sync *lll_adv_sync;
+	struct lll_adv_iso *lll_adv_iso;
 	struct node_rx_pdu *node_rx;
+	struct ll_adv_iso *adv_iso;
+	struct lll_adv *lll_adv;
 	uint32_t ret;
+	uint8_t err;
 
 	adv_iso = ull_adv_iso_get(big_handle);
-
 	if (!adv_iso) {
 		return BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER;
 	}
 
+	lll_adv_iso = &adv_iso->lll;
+	lll_adv = lll_adv_iso->adv;
```

Comment:
Do we need a check for `lll_adv_iso` not being NULL?

Reply:
No. `lll_adv_iso` is reference to a static structure member. see https://github.com/zephyrproject-rtos/zephyr/blob/38a9e945ecac3367d4d973062d434990defc02ad/subsys/bluetooth/controller/ll_sw/ull_adv_types.h#L66